### DISCUSSION TITLE
Question: What are your plans for model training?

### DISCUSSION THREAD

**Discussion by malteos** [LEL-A/GerAlpacaDataCleaned#9] [Q&A]
Hi Stefan,

Thanks for sharing the dataset. I was about to do the same. Do have any concrete plans for training models on this data? My plan was to fine tune: https://huggingface.co/malteos/bloom-6b4-clp-german 

Best
Malte

**Comment by PhilipMay** [comment]
Hi @malteos ,
yes. We plan this and much more! 😀
We have documented our goals here: https://github.com/LEL-A/doc#readme

If you are interested to join and contribute: we would be happy to welcome you in our community. 🚀 👍🏽

**Comment by PhilipMay** [comment]
PS: @malteos our plan was exectly what you suggest here - use this dataset and your Bloom model as a first POC so to say. But then later to train a LLaMA style model with more Euro languages. 

And from there go with multilingual Alpaca training.

**Comment by stefan-it** [comment]
Hi @malteos,

thanks for reaching out! As @PhilipMay mentioned, your model will serve as backbone for our initial experiments with the Alpaca fine-tuning. 

I can also think about train the model longer on newer corpora e.g. the lastest OSCAR release :hugs:

**Comment by malteos** [comment]
> Hi @malteos,
> 
> thanks for reaching out! As @PhilipMay mentioned, your model will serve as backbone for our initial experiments with the Alpaca fine-tuning.
> 
> I can also think about train the model longer on newer corpora e.g. the lastest OSCAR release 🤗

Haha :D That is also currently in progress. There will be another version of my BLOOM-CLP-German trained on more tokens very soon. We are also preparing additional OSCAR releases ;) Are you available for a call anytime next week?

**Comment by PhilipMay** [comment]
Sure. Good idea! I will send an invite next week. 👍

**Comment by PhilipMay** [comment]
I have sent a proposed date to your mail address @malteos :
https://ostendorff.org/contact/
